src/download.rs

```diff
@@ -73,7 +73,7 @@ pub fn download_love(version: LoveVersion, platform: Platform, bitness: Bitness)
                 .unwrap_or_else(|_| panic!("Could not get archive file by index '{}'", i));
             let mut outpath = output_file_path.clone();
             outpath.pop();
-            outpath.push(file.sanitized_name());
+            outpath.push(file.mangled_name());
```

Comment:
I didn't know that `sanitized_name` was deprecated. It looks like `enclosed_name` is generally the more recommended option over `mangled_name`. Would this be an appropriate place to use it here as well, or is there any specific reason to prefer `mangled_name`?

https://docs.rs/zip/0.5.13/zip/read/struct.ZipFile.html#method.enclosed_name

Reply:
The compiler suggested I use `mangled_name`, but you are absolutely correct, the documentation states that one should use `enclosed_name`, so let's go with that.,